api/app/signals/apps/api/generics/mixins.py

```diff
@@ -97,14 +98,15 @@ def _set_write_once_fields(self, extra_kwargs):
         return extra_kwargs
 
 
-class NearAmsterdamValidatorMixin:
+class WithinBoundingBoxValidatorMixin:
 
     def validate_geometrie(self, value):
-        fail_msg = 'Location coordinates not anywhere near Amsterdam. (in WGS84)'
+        fail_msg = 'The location coordinates (WS84) are not inside the bounding box we are accepting signals for.'
 
-        lat_not_in_adam_area = not 50 < value.coords[1] < 55
-        lon_not_in_adam_area = not 1 < value.coords[0] < 7
+        lat_not_in_bounding_box = not settings.BOUNDING_BOX_MIN[0] <= value.coords[1] <= settings.BOUNDING_BOX_MAX[0]
```

Comment:
SIA/Signalen uses (lon, lat) coordinate order, please conform. See also the comment on `api/app/signals/settings/base.py`.

Reply:
Fixed